modules/rest-api/src/main/java/org/apache/ignite/internal/rest/api/cluster/ClusterState.java

```diff
@@ -34,24 +36,31 @@
  */
 @Schema(description = "Information about current cluster state.")
 public class ClusterState {
-    @Schema(description = "List of cluster management group nodes. These nodes are responsible for maintaining RAFT cluster topology.")
+    @Schema(description = "List of cluster management group nodes. These nodes are responsible for maintaining RAFT cluster topology.",
+            requiredMode = RequiredMode.REQUIRED)
     @IgniteToStringInclude
     private final Collection<String> cmgNodes;
 
-    @Schema(description = "List of metastorage nodes. These nodes are responsible for storing RAFT cluster metadata.")
+    @Schema(description = "List of metastorage nodes. These nodes are responsible for storing RAFT cluster metadata.",
+            requiredMode = RequiredMode.REQUIRED)
     @IgniteToStringInclude
     private final Collection<String> msNodes;
 
-    @Schema(description = "Version of Apache Ignite that the cluster was created on.")
+    @Schema(description = "Version of Apache Ignite that the cluster was created on.", requiredMode = RequiredMode.REQUIRED)
     private final String igniteVersion;
 
-    @Schema(description = "Unique tag that identifies the cluster.")
+    @Schema(description = "Unique tag that identifies the cluster.",
+            requiredMode = RequiredMode.REQUIRED)
     private final ClusterTag clusterTag;
 
     @Schema(description = "IDs the cluster had before.")
     @IgniteToStringInclude
     private final @Nullable List<UUID> formerClusterIds;
 
+    @Schema(description = "Cluster status.",
+            requiredMode = RequiredMode.REQUIRED)
+    private final ClusterStatus clusterStatus;
```

Comment:
It seems that entities from different domains are mixed here. This `ClusterState` class represents the CMG cluster state that is the state stored in the CMG related to the cluster itself. But `ClusterStatus` is some volatile thing, it might change during runtime however it wants, even if the CMG cluster state does not change.

I believe we should not mix them together. Could a distinct API endpoint (and CLI command) be added to access this volatile information?